/*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

#pragma once

namespace facebook {
namespace terragraph {
namespace mobd {

bool unitTest();

} // namespace mobd
} // namespace terragraph
} // namespace facebook
